### DISCUSSION TITLE
my client.realm wont work anymore in the latest version

### DISCUSSION THREAD

**Discussion by MOOMAIV** [ppy/osu#30371] [Q&A] [answered]
I went to install osu and accidentally installed a local/dev build from a aur. Now my client.realm that has all my songs ect refuses to work with the flatpack or app image version.

This is the version i downloaded https://aur.archlinux.org/packages/osu-lazer-git

Log/client.realm
[logs⁄.realm.zip](https://github.com/user-attachments/files/17452713/logs.realm.zip)

**Comment by bdach** [comment, marked as answer]
There is nothing that can be done at this stage. Please wait for the next release with the relevant realm schema version bump.

Actual dev builds that are not "from an aur" have explicit protections against this sort of thing happening. This is what happens when you let people redistribute software without knowing how it works.
